src/main/java/ladder/controller/LadderGameController.java

```diff
@@ -51,17 +71,53 @@ private Height generateHeight() {
 
     private Ladder generateLadder(Players players, Height height) {
         LadderGenerator ladderGenerator = new LadderGenerator(lineCreateDecider);
-        return ladderGenerator.generateLadder(players.size(), height);
+        return ladderGenerator.generateLadder(players.getSize(), height);
+    }
+
+    private void showLadderGame(Players players, Ladder ladder, Rewards rewards) {
+        outputView.printLadderResultMessage();
+        showPlayers(players);
+        showLadder(ladder);
+        showRewards(rewards);
     }
 
-    private void showResult(Players players, Ladder ladder) {
+    private void showPlayers(Players players) {
         outputView.printPlayerNames(players.getPlayers().stream()
                 .map(Player::getPlayerName)
                 .collect(Collectors.toList()));
+    }
+
+    private void showLadder(Ladder ladder) {
+        ladder.getRows().forEach(row -> outputView.printRow(row.getPoints()));
+    }
+
+    private void showRewards(Rewards rewards) {
+        outputView.printRewards(rewards.getRewards().stream()
+                .map(Reward::getReward)
+                .collect(Collectors.toList()));
+    }
 
-        List<Row> rows = ladder.getRows();
-        for (Row row : rows) {
-            outputView.printRow(row.getPoints());
+    private void showResultBoard(ResultBoard resultBoard) {
+        String option = inputView.readPlayerChoice();
+
+        while (!option.equals(Command.LOOK_ALL_PLAYERS_RESULT.getCommand())) {
```

Comment:
option이 null이면 어떻게 될까요?

Reply:
별다른 입력없이 바로 엔터를 치는 상황을 말씀하시는 것이라면, 현재 코드에서는 정상 작동을 하고 있습니다!
혹시 따로 `null` 값을 처리하는 메소드를 분리해서 쓰는게 좋을까요?

Reply:
지금은 정상 작동하지만, 비정상적인 입력으로 인해 option에 null이 들어오면 NPE가 발생할 수 있어요.
그래서 동등 비교는 다음과 같이 작성하는 게 안전해요.

```suggestion
        while (!Command.LOOK_ALL_PLAYERS_RESULT.getCommand().equals(option)) {
```

Reply:
오 그런 경우에는 option의 equals를 호출 할 수 없겠군요! 생각치 못했던 포인트였는데 감사합니다!!